### PR TITLE
Update Power Workspace/s and Datacenter/s  DataSource Documentation 

### PR DESCRIPTION
**Pull request by michaelkad**
- Update doc for get and getall workspace and datacenter

### Community Note

* Please vote on this pull request by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original pull request comment to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for pull request followers and do not help prioritize the request

Relates OR Closes #0000

Output from acceptance testing:

```
$ make testacc TESTARGS='-run=TestAccXXX'

...
```

### CODE REVIEW

**Comment by michaelkad** [comment]
@hkantare @yussufsh  Could you help with reviewing this PR please?

**Comment by yussufsh** [comment]
> @hkantare @yussufsh Could you help with reviewing this PR please?

There are some pending comments above.

**Comment by michaelkad** [comment]
@yussufsh I do not think there has been any comments on this PR. If you were in the middle of one, please do resubmit.

**Review by yussufsh — CHANGES_REQUESTED** [review]

**Comment by yussufsh** [comment]
> @yussufsh I do not think there has been any comments on this PR. If you were in the middle of one, please do resubmit.

You are right.. Sorry :-(

File: website/docs/d/pi_datacenter.html.markdown [hunk 1]
```diff
@@ -45,20 +45,16 @@ Review the argument references that you can specify for your data source.
...
 
 - `pi_datacenter_location` - (Map) Datacenter location.
 
     Nested schema for `pi_datacenter_location`:
   - `region` - (String) The Datacenter location region zone.
   - `type` - (String) The Datacenter location region type.
   - `url`- (String) The Datacenter location region url.
-- `pi_datacenter_status` - (String) The Datacenter status, `ACTIVE`,`MAINTENANCE` or `DOWN`.
-- `pi_datacenter_type` - (String) The Datacenter type, `Public Cloud` or `Private Cloud`.
+- `pi_datacenter_status` - (String) The Datacenter status, `active`,`maintenance` or `down`.
```
[the remark this answers is not in the document]
Comment (michaelkad) [reply website/docs/d/pi_datacenter.html.markdown:59]: You are right,  I will update the go.mod 
They are the same however, they were updated to lowercase. 
https://github.com/powervs-ibm/power-go-client/blob/782d50a27da76f31d0cc1c88cf2c6a63c9f885fa/power/models/datacenter.go#L36

File: website/docs/d/pi_datacenters.html.markdown [hunk 2]
```diff
@@ -42,20 +42,16 @@ In addition to all argument reference list, you can access the following attribu
...
   - `pi_datacenter_location` - (Map) Datacenter location.
 
     Nested schema for `pi_datacenter_location`:
     - `region` - (String) The Datacenter location region zone.
     - `type` - (String) The Datacenter location region type.
     - `url`- (String) The Datacenter location region url.
-  - `pi_datacenter_status` - (String) The Datacenter status, `ACTIVE`,`MAINTENANCE` or `DOWN`.
-  - `pi_datacenter_type` - (String) The Datacenter type, `Public Cloud` or `Private Cloud`.
+  - `pi_datacenter_status` - (String) The Datacenter status, `active`,`maintenance` or `down`.
+  - `pi_datacenter_type` - (String) The Datacenter type, `off-premises` or `on-premises`.
```
[the remark this answers is not in the document]
Comment (michaelkad) [reply website/docs/d/pi_datacenters.html.markdown:57]: You are right,  I will update the go.mod

**Comment by michaelkad** [comment]
> > @yussufsh I do not think there has been any comments on this PR. If you were in the middle of one, please do resubmit.
> 
> You are right.. Sorry :-(

You are good.  I will update go.mod

**Review by yussufsh — APPROVED** [review]
lgtm
@hkantare please help merge this one.

**Comment by hkantare** [comment]
fix the conflicts

**Comment by michaelkad** [comment]
> fix the conflicts

@hkantare The conflict has been resolved.

**Pull request merged**